manim/mobject/coordinate_systems.py

```diff
@@ -154,6 +154,9 @@ def __init__(self, **kwargs):
         self.axes = VGroup(self.x_axis, self.y_axis)
         self.add(*self.axes)
         self.shift(self.center_point)
+        ynums = self.y_axis.numbers
+        for num in range(len(ynum)):
+            self.y_axis.numbers[num].rotate(-90 * DEGREES)
```

Comment:
We actually don't need to introduce this local variable there:

```suggestion
        for num in self.y_axis.numbers:
            num.rotate(-90 * DEGREES)
```

Reply:
Wait, this will also not work: `y_axis` does not have an attribute `numbers`, so your suggestion cannot work. Please update your PR.